lib/wavefile/chunk_readers/smpl_chunk_reader.rb

```diff
@@ -0,0 +1,118 @@
+module WaveFile
+  module ChunkReaders
+    # Internal
+    class SmplChunkReader < BaseChunkReader    # :nodoc:
+      # Public: Returns the ID of the manufacturer
+      # See the list at https://www.midi.org/specifications-old/item/manufacturer-id-numbers
+      attr_reader :manufacturer_id
+
+      # Public: Returns the ID of the product
+      attr_reader :product_id
+
+      # Public: Returns the length of each sample in nanoseconds
+      attr_reader :sample_duration
+
+      # Public: Returns the MIDI note number of the sample (0-127)
+      attr_reader :unity_note
+
+      # Public: Returns the fraction of a semitone up from the specified MIDI unity note field.
+      # A value of 0x80000000 means 1/2 semitone (50 cents) and a value of 0x00000000 means no fine tuning between semitones.
+      # - https://sites.google.com/site/musicgapi/technical-documents/wav-file-format
+      # Integer
+      attr_reader :pitch_fraction
+
+      # Public: Returns the SMPTE format (0, 24, 25, 29 or 30)
+      attr_reader :smpte_format
+
+      # Public: Returns the SMPTE time offset.
+      # This value uses a format of 0xhhmmssff where hh is a signed value that specifies the number of hours (-23 to 23),
+      # mm is an unsigned value that specifies the number of minutes (0 to 59), ss is an unsigned value that specifies
+      # the number of seconds (0 to 59) and ff is an unsigned value that specifies the number of frames (0 to -1).
+      # - https://sites.google.com/site/musicgapi/technical-documents/wav-file-format
+      attr_reader :smpte_offset
```

Comment:
It might be better to create a data structure that contains the sub fields (hours, minutes, etc.), instead of making the user parse this data out of an integer. It seems similar to the `Duration` object, but unfortunately not quite the same? (I'm not very familiar with SMPTE time codes).